src/server.py

```diff
@@ -429,10 +406,7 @@ def run_seta_details_query():
         today = datetime.now()
         date = today.strftime("%Y-%m-%d")
 
-    db = create_db_connnection()
-    cursor = db.cursor()
-    query = "select jobtype from seta where date='%s 00:00:00'" % date
-    cursor.execute(query)
+    query = session.query(Seta.jobtype).filter(Seta.date == date).all()
```

Comment:
I usr this I use this http://0.0.0.0:8157/data/setadetails/?date=2016-8-7&priority=high to test it and get:

<img width="1255" alt="screen shot 2016-08-08 at 8 51 59 pm" src="https://cloud.githubusercontent.com/assets/2166227/17480708/bebe3106-5dab-11e6-9423-eb7c1420d666.png">

So I think it's ok :)